requirements/dev.txt

```diff
@@ -31,7 +31,7 @@ code-annotations==1.3.0
     # via edx-lint
 commonmark==0.9.1
     # via rich
-cryptography==38.0.4
+cryptography==39.0.0
```

Comment:
Major breaking changes regarding `openssl`, `hashes` and `required mac version`. For details see https://cryptography.io/en/latest/changelog/#v39-0-0